Dockerfile.stable

```diff
@@ -20,15 +24,20 @@ RUN mkdir -p /root/chrome-driver &&\
 ADD ./xstartup.sh /xstartup.sh
 RUN chmod 775 /xstartup.sh
 # selenium
-ADD http://selenium-release.storage.googleapis.com/$SELENIUM_VERSION/selenium-server-standalone-$SELENIUM_VERSION.0.jar \
+# TODO: make better url scheme
+ADD http://selenium-release.storage.googleapis.com/3.12/selenium-server-standalone-$SELENIUM_VERSION.jar \
```

Comment:
Why hardcode 3.12 instead of SELENIUM_VERSION?

Reply:
the cutoff is incorrect, so this is a partial solution eventually there will be a major/minor + a patchlevel var